### PR TITLE
Bug fixes for ustruct Guccione model and follower pressure load

### PR DESCRIPTION
**Pull request by aabrown100-git**
## Current situation
Slow nonlinear convergence for in ustruct/LV_Guccione_active. See issue: https://github.com/SimVascular/svFSIplus/issues/197

## Release Notes 
Fix typos in Guccione material model used for ustruct, and follower pressure load implementation for ustruct. These are now consistent with the corresponding lines in svFSI (Fortran)

I made these changes to match the following lines in svFSI (Fortran) code
https://github.com/SimVascular/svFSI/blob/7bdb74cdd517310d6bf32c0bb7d42072aeac8658/Code/Source/svFSI/MATMODELS.f#L809

https://github.com/SimVascular/svFSI/blob/7bdb74cdd517310d6bf32c0bb7d42072aeac8658/Code/Source/svFSI/MATMODELS.f#L812

https://github.com/SimVascular/svFSI/blob/7bdb74cdd517310d6bf32c0bb7d42072aeac8658/Code/Source/svFSI/USTRUCT.f#L1320

Unfortunately, after making these changes, the convergence for ustruct/LV_Guccione_active is still not good. There must be another bug in the tangent somewhere, but I've gone through the Guccione model in get_pk2cc_dev() several times and can't find anything.

## Testing
**Warning: these changes produced different results for ustruct/LV_Guccione_active than the existing reference solution. I updated the reference solution for this test.**

**Changes do not significantly improve convergence of ustruct/LV_Guccione_active**. There are likely other bugs. However, I think these are clearly bugs, so it makes sense to fix them now.

## Code of Conduct & Contributing Guidelines 
- [x] I agree to follow the [Code of Conduct](https://github.com/SimVascular/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/SimVascular/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

**Comment by mrp089** [comment]
Thank you for digging into this, @aabrown100-git! We're down to 17 Newton iterations from 19. Every bug counts! I'll let @ktbolt review this.

**Comment by mrp089** [comment]
@ktbolt, can you review this PR? Thank you!

File: Code/Source/svFSI/mat_models.cpp [hunk 1]
```diff
@@ -690,10 +690,10 @@ void get_pk2cc_dev(const ComMod& com_mod, const CepMod& cep_mod, const dmnType&
            2.0*g3*(Es(0,1)*RmRm.slice(3) + Es(0,2)*RmRm.slice(5));
 
       auto CCb = 2.0*ten_dyad_prod(Sb, Sb, nsd);
-      Sb += Sb * r2;
+      Sb = Sb * r2;
 
       // Fiber reinforcement/active stress
-      Sb += Sb + Tfa*mat_dyad_prod(fl.col(0), fl.col(0), nsd);
+      Sb = Sb + Tfa*mat_dyad_prod(fl.col(0), fl.col(0), nsd);
```
Comment (ktbolt) [inline Code/Source/svFSI/mat_models.cpp:696]: I believe that `Sb += Tfa*mat_dyad_prod(fl.col(0), fl.col(0), nsd);` is more efficient, does not create an intermediate Array object.

Comment (aabrown100-git) [reply Code/Source/svFSI/mat_models.cpp:696]: Will do!

**Review by ktbolt — APPROVED** [review]
Looks good!

**Pull request merged**